audb/core/info.py

```diff
@@ -11,25 +11,32 @@
     dependencies,
     latest_version,
 )
+from audb.core.load import (
```

Comment:
The following imports we be removed now:

```
import os
import tempfile
from audb.core import define
from audb.core.utils import lookup_backend
```

Reply:
Thanks for spotting this, sometimes `flake8` finds this for me, sometimes not.

I removed it now.